examples/gnrc_networking/Makefile

```diff
@@ -34,7 +34,8 @@ USEMODULE += ps
 USEMODULE += netstats_l2
 USEMODULE += netstats_ipv6
 USEMODULE += netstats_rpl
-
+USEMODULE += gnrc_ipv6_ext
+USEMODULE += gnrc_rpl_hop
```

Comment:
please adjust the dependencies for `gnrc_rpl_hop` to include `gnrc_ipv6_ext` automagically